### ISSUE TITLE
The effect is not ideal

### ISSUE THREAD

**Issue by FiLafs** [YChenL/DS-TDNN#2]
I set dim=512 and trained 80 epochs, but the final EER was only 1.31%. After using AS norm, it was also around 1.15%, which did not reach the EER of 0.9% in the paper. Additionally, the size of the model was larger than that in the paper. What is the reason for this?

**Comment by wcqy-ye** [comment]
> I set dim=512 and trained 80 epochs, but the final EER was only 1.31%. After using AS norm, it was also around 1.15%, which did not reach the EER of 0.9% in the paper. Additionally, the size of the model was larger than that in the paper. What is the reason for this?

May I ask if you were able to successfully replicate the results from the paper? I have attempted to replicate them without performing data augmentation, but the EER consistently hovers around 1.4%. I would be very grateful for your assistance if possible. Thank you.

**Comment by YChenL** [comment]
The paper is still under review. When the paper is accepted, more technical details will be reported in the appendix of the paper. At the same time, the pre-trained model parameters will also be provided in this repo, which can achieve the results reported in this paper.
